Makefile

```diff
@@ -327,6 +327,15 @@ auto-updates: require-version require-s3cmd require-gh-token require-ruby
 
 release: require-version require-s3cmd require-wget require-lantern-binaries require-release-track release-prod copy-beta-installers-to-mirrors invalidate-getlantern-dot-org upload-aab-to-play
 
+# Note from https://github.com/wlynxg/anet?tab=readme-ov-file#how-to-build-with-go-1230-or-later
```

Comment:
@atavism , with the new go we have to pass a new linker flash, Did you test with these new changes? I did that changes before and somehow proxy was not working